ocfweb/account/templates/chpass/index.html

```diff
@@ -1,20 +1,48 @@
 {% extends 'base.html' %}
-
-{% block title %}Change Password{% endblock %}
+{% load bootstrap %}
 
 {% block content %}
-    {% if error %}
-        <ul class="errorlist">
-            <li>{{error}}</li>
-        </ul>
-    {% endif %}
+    <div class="ocf-content-block">
+        <div class="row">
+            <div class="col-sm-8">
+                {% if error %}
+                    <div class="alert alert-danger" role="alert">
+                        {{ error }}
+                    </div>
+                {% endif %}
+
+                <form method="post" action="{% url 'change_password' %}">
+                    {% csrf_token %}
+                    <div style="max-width: 300px;">
+                        {{ form|bootstrap }}
+                    </div>
+                    <input class="btn btn-primary" type="submit" value="Change Password" />
+                    <a class="btn btn-default" href="{% url 'calnet_logout' %}">Cancel</a>
+                </form>
+            </div>
+            <div class="col-sm-4">
+                <h3>Password requirements</h3>
+                <p>Your password should be:</p>
+                <ul>
+                    <li>At least 8 characters long (preferably much longer)</li>
+                    <li>Reasonably complex</li>
+                    <li>Not based on your username</li>
+                    <li>Not based on a single dictionary word</li>
+                </ul>
 
-    <form method="post" action="{% url 'change_password' %}" id="change-password-form">
-        {% csrf_token %}
-        {{ form.as_p }}
-        <input type="submit" value="Change Password" />
-    </form>
+                <p>
+                    <strong>Suggestion!</strong>
+                    <a href="https://xkcd.com/936/">String together 4 words
```

Comment:
I'm genuinely concerned it might not be a good idea to endorse said password generation method *right on this page*, as pretty much any password method you can pick has some level of contention about it and I don't want to us to be responsible if someone's password gets dictionary cracked.